lua/possession/commands.lua

```diff
@@ -128,7 +128,11 @@ end
 
 ---@param name? string
 function M.load_cwd(name)
-    name = name_or(name, paths.cwd_session_name)
+    local last = function()
+        return get_last(get_sessions_for_dir(vim.fn.getcwd()))
+    end
+
+    name = name_or(name, last)
```

Comment:
We're not validating that the user provided name matches a cwd session. If it matches any session it will load. I think this is fine. 

The autocomplete only shows cwd sessions.

Reply:
Seems fine